packages/manager/apps/telecom/src/components/notification/notification.component.js

```diff
@@ -213,12 +213,18 @@ angular.module('managerApp').component('notificationList', {
       // get the SMS accounts
       OvhApiSms.Aapi()
         .query({})
-        .$promise.then(
-          (data) => {
-            self.accounts = data;
-          },
-          (err) => self.processError(err),
-        );
+        .$promise.then((data) => {
+          self.accounts = data;
+          // get the SMS remaining credits by accounts
+          OvhApiSms.Aapi()
```

Comment:
Instead of calling the same 2API twice, why not considering using something like:

```js
iceberg('/sms')
  .query()
  .expand('CachedObjectList-Pages')
  .execute()
  .$promise.then(({ data: smsDetails }) => {
    self.accounts = _.map(smsDetails, 'name');
    self.smsDetails = smsDetails;
  });
```

It will ended up with a single API call ~227ms instead of having two call (~212ms +~ 267ms).

Please let me know what do you think about it:)

Reply:
Good idea, I didn't know about this possibility.

Reply:
It's part of the package called `@ovh-ux/ng-ovh-api-wrapper`.

There is a service called [`iceberg`](https://github.com/ovh-ux/ng-ovh-api-wrappers/blob/master/src/index.js#L35) that you can use.

A documentation can be found here: https://github.com/ovh-ux/ng-ovh-api-wrappers#iceberg

Hope you find this useful!